src/openafval/trash/models.py

```diff
@@ -0,0 +1,263 @@
+        verbose_name=_("fraction"),
+        help_text=_("The fraction of the container."),
+        on_delete=models.CASCADE,
+        null=True,
+    )
+    type = models.CharField(
+        _("type"),
+        max_length=120,
+        help_text=_("The type of the container."),
+        unique=True,
+    )
+    description = models.TextField(
+        _("description"),
+        help_text=_("The description of the container."),
+        blank=True,
+    )
+
+    class Meta:  # pyright: ignore
+        verbose_name = _("container type")
+        verbose_name_plural = _("container types")
+
+    def __str__(self):
+        return f"{self.type} - {self.description}"
+
+
+class Container(models.Model):
+    id: int
+    type = models.ForeignKey(
+        ContainerType,
+        verbose_name=_("type"),
+        help_text=_("The type of the container."),
+        on_delete=models.CASCADE,
+        null=True,
+    )
+    identifier = models.CharField(
+        _("identifier"),
+        help_text=_("Imported identifier for the Container."),
+        max_length=255,
+        unique=True,
+    )
+    is_collection_container = models.BooleanField(
+        _("is collection container"),
+        help_text=_("Whether the container has a collection container."),
+        default=False,
+    )
+    has_key = models.BooleanField(
+        _("has a key"),
+        help_text=_("Whether the container has a key."),
+        default=False,
+    )
+
+    class Meta:  # pyright: ignore
+        verbose_name = _("container")
+        verbose_name_plural = _("containers")
+
+    def __str__(self):
+        return self.identifier
+
+
+class Emptying(models.Model):
+    id: int
+    container = models.ForeignKey(
+        Container,
+        verbose_name=_("container"),
+        help_text=_("The container which was emptied."),
+        on_delete=models.CASCADE,
+    )
+    entity_object_management = models.ManyToManyField(
+        EntityObjectManagement,
+        verbose_name=_("entity/object management"),
+        help_text=_("The entity/object connection that this emptying is tied to."),
+    )
+    identifier = models.CharField(
+        _("identifier"),
+        help_text=_("Imported identifier for the Emptying."),
+        max_length=255,
+        unique=True,
+    )
+    weight_dispersed = models.CharField(
+        _("weight dispersed"),
+        help_text=_("The dispersed weight of the container that got dumped."),
+        max_length=20,
+        blank=True,
+    )
+    weight_none_dispersed = models.CharField(
+        _("weight none dispersed"),
+        help_text=_("The none dispersed weight of the container that got dumped."),
+        max_length=20,
+        blank=True,
+    )
+    amount = models.IntegerField(
+        _("amount"),
+        help_text=_("The amount of times the container needed to be dumped to empty."),
+        default=1,
+    )
+    date = models.DateField(
+        _("date"),
+        help_text=_("The date of the emptying."),
+    )
```

Comment:
We can parse the `date` from `datetime` when needed, it's redundant to have both (though I see it's in the CSV for some reason, so I'm not sure)